addons/mass_mailing/static/src/js/mailing_mailing_view_form_full_width.js

```diff
@@ -105,11 +105,11 @@ export class MassMailingFullWidthViewController extends formView.Controller {
      * @private
      */
     _onToggleFullscreen() {
+        const isFullscreen = this._isFullScreen();
```

Comment:
See general response above.

Reply:
Ok then, makes sense.

Maybe add a small comment above this line?
Something like:
`# the actual toggling is done by the wysiwyg, here we just read the current fullscreen status and adapt accordingly`

Also, runbot seems mad, maybe a rebase/rebuild?

Reply:
Hmm, it's not visible in the commit on github but this method has a docstring that says `Switch "scrolling modes" on toggle fullscreen mode` so I feel like it would be redundant.

Test failed randomly, I rebuilt :)